routing/pb/routing_state.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package routing.pb;
+
+message RoutingStateRecord {
+    message AddressInfo {
+        bytes multiaddr = 1;
+    }
+
+    bytes peer_id = 1;
```

Comment:
we usually camelCase those.

Reply:
I noticed that, but someone (maybe @raulk; I can't seem to find the original comment now) mentioned that snake_case is [recommended by google](https://developers.google.com/protocol-buffers/docs/style#message-and-field-names) - it gets mapped to camelCase in the generated code (except langs like c++ and maybe python).

I don't really care either way, but it does bug me that I'm adding code that isn't consistent with the rest of our protos... if we plan to change everything to snake_case though, I'd rather start doing it now.

Reply:
found the comment I was referring to above; it's from the RFC: https://github.com/libp2p/specs/pull/217#discussion_r345124996

Reply:
all our protobufs use camelCase though.

Reply:
yeah. I think I'll change it back to camelCase & leave it as snake case in the RFC. if we decide we want go-libp2p to use snake case, we should convert them all in one pass, so they stay consistent.